backend/tracim_backend/migration/versions/a074395d3caf_add_subtype_in_events_table.py

```diff
@@ -0,0 +1,26 @@
+"""add subtype in events table
+
+Revision ID: a074395d3caf
+Revises: 3c92f57c52b2
+Create Date: 2020-05-26 11:30:36.254925
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "a074395d3caf"
+down_revision = "3c92f57c52b2"
+
+
+def upgrade():
+    with op.batch_alter_table("events") as batch_op:
+        batch_op.add_column(sa.Column("entity_subtype", sa.String(), nullable=True))
+    # NOTE SG 2020-05-26: subtype values are left to NULL as we do not have any production
+    # database running with the events table.
```

Comment:
What does this imply for developer database that used the previous implementation of TLM, and do try to use the same database updated ?

Reply:
This imply that content events/messages previously saved in the database won't have their content-type added when queried through the HTTP API.
As we only have some developer databases, the easier way is for them to delete/recreate their database.

Production database migration will be OK as the events table didn't exist in 2.7.x.